recipes-support/simple-http-server/files/simple-http-server.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+pushd /etc/hosted_files
```

Comment:
@SaliniVenate Can you also check the write permission to that dir. CSD need to be able to copy files(camera-definition/image/video) in that location